admin/differ/DifferModel.js

```diff
@@ -270,25 +306,22 @@ export class DifferModel  {
 
     valueRenderer(v) {
```

Comment:
I think this should be moved out of here and be part of the ConfigModel column def